### PR TITLE
Putting our foot down for the Nth time (a.k.a. Indentation IV: electric hullabaloo)

### PR DESCRIPTION
**Pull request by felix-hilden**
### Description
From #2798: I'd like to include a FAQ entry about our refusal for tab support, and a short statement about the accessibility issue. TL;DR: never gonna happen, but we describe an alternate path that a visually impaired developer might use for a better experience with the Python ecosystem.

But is it too out of scope?

### Checklist - did you ...
- [x] Add new / update outdated documentation?

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ The most common questions and issues users face are aggregated to this FAQ.
 :class: this-will-duplicate-information-and-it-is-still-useful-here
 ```
 
+## Why spaces? I prefer tabs
+
+PEP 8 recommends spaces over tabs, and they are used by most of the Python community.
+There are no options to configure the indentation style, and requests for any such
```
Comment (JelleZijlstra) [inline docs/faq.md:14]:
```suggestion
_Black_ provides no options to configure the indentation style, and requests for any such
```
There are options, just not here.

**Comment by JelleZijlstra** [comment]
Now it needs to be run through prettier, sorry. Feel free to merge after you fix that!

File: docs/faq.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ The most common questions and issues users face are aggregated to this FAQ.
 :class: this-will-duplicate-information-and-it-is-still-useful-here
 ```
 
+## Why spaces? I prefer tabs
+
+PEP 8 recommends spaces over tabs, and they are used by most of the Python community.
+_Black_ provides no options to configure the indentation style, and requests for any such
```
Comment (felix-hilden) [inline docs/faq.md:14]:
```suggestion
_Black_ provides no options to configure the indentation style, and requests for such
```

**Comment by felix-hilden** [comment]
Yay, win for wording *and* prettier!

**Pull request merged**